src/components/LiveGraphPanel.tsx

```diff
@@ -92,7 +92,11 @@ const LiveGraphPanel = ({
             <Text>
               <FormattedMessage id={disconnectedTextId} />
             </Text>
-            <Button variant="primary" onClick={handleConnectOrReconnect}>
+            <Button
+              variant="primary"
+              onClick={handleConnectOrReconnect}
+              aria-label={intl.formatMessage({ id: "connect-action-aria" })}
```

Comment:
Introduced aria-label for button on the basis that we don't want to change the button text itself?